modules/fbs-core/api/src/main/resources/migrations/01_init.sql

```diff
@@ -24,8 +24,9 @@ USE `fbs` ;
 DROP TABLE IF EXISTS `fbs`.`course` ;
 
 CREATE TABLE IF NOT EXISTS `fbs`.`course` (
-                                              `course_id` INT NOT NULL AUTO_INCREMENT,
-                                              `name` VARCHAR(100) NOT NULL,
+    `course_id` INT NOT NULL AUTO_INCREMENT,
+    `semester` INT NOT NULL DEFAULT 0, -- 0 == not specified
+    `name` VARCHAR(100) NOT NULL,
```

Comment:
Changes in the database scheme must be located in a new migration file. See https://easytutor.mni.thm.de:3000/de/fbs/migration for more information.